.codeclimate.yml

```diff
@@ -10,4 +10,4 @@ exclude_paths:
 - "**/*.md"
 - "Dockerfile"
 - "bin/fixme"
-- "tests/**"
+- "test/**"
```

Comment:
Good philosophical question.

I think it's a good use case for per-engine excludes.

TODOs are equally issues when included in tests IMO, but maybe we don't care as much about checks for duplication and complexity in that code.